src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/MethodBuilderImpl.cs

```diff
@@ -106,15 +138,36 @@ protected override void SetImplementationFlagsCore(MethodImplAttributes attribut
             _methodImplFlags = attributes;
         }
         protected override void SetSignatureCore(Type? returnType, Type[]? returnTypeRequiredCustomModifiers, Type[]? returnTypeOptionalCustomModifiers, Type[]? parameterTypes,
-            Type[][]? parameterTypeRequiredCustomModifiers, Type[][]? parameterTypeOptionalCustomModifiers) => throw new NotImplementedException();
+            Type[][]? parameterTypeRequiredCustomModifiers, Type[][]? parameterTypeOptionalCustomModifiers)
```

Comment:
It may be nice to add TODO about missing support for parameterTypeRequiredCustomModifiers/parameterTypeOptionalCustomModifiers